rules/naming/src/RectorNamingInflector.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Naming;
+
+use Doctrine\Inflector\Inflector;
+use Doctrine\Inflector\InflectorFactory;
+use Doctrine\Inflector\Language;
+use Doctrine\Inflector\Rules\Pattern;
+use Doctrine\Inflector\Rules\Patterns;
+use Doctrine\Inflector\Rules\Ruleset;
+use Doctrine\Inflector\Rules\Substitutions;
+use Doctrine\Inflector\Rules\Transformation;
+use Doctrine\Inflector\Rules\Transformations;
+
+final class RectorNamingInflector
+{
+    public function build(): Inflector
+    {
+        // @see https://www.doctrine-project.org/projects/doctrine-inflector/en/2.0/index.html
+        return InflectorFactory::createForLanguage(Language::ENGLISH)
+            ->withSingularRules(
+                new Ruleset(
+                    new Transformations(new Transformation(new Pattern('/^packagesData$/i'), 'packageData')),
```

Comment:
BTW, these exceptions could be a rule setting in the future :)

Reply:
or... Set setting.

Reply:
Could you add test for this class? Just to confirm its doing the job right

Reply:
Could you cover " anythingData" in tests too?

Reply:
In order to make "anythingData" work, I had to change the logic.